### DISCUSSION TITLE
import chigger failed

### DISCUSSION THREAD

**Discussion by hugary1995** [idaholab/moose#18032] [Q&A Tools] [answered]
Hi all,

I wanted to use chigger to generate some animations, but I can't seem to load chigger even though I have moose environment activated. What am I missing?

Thanks,
Gary

**Comment by hugary1995** [comment, marked as answer]
I forgot to add moose/python to PYTHONPATH...

**Reply by aeslaughter** [reply]
I am glad I am not the only one, I do this all the time. 🤣
